extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/security/HttpSecurityRecorder.java

```diff
@@ -256,8 +256,13 @@ public FormAuthenticationMechanism get() {
                 String loginPage = form.loginPage.startsWith("/") ? form.loginPage : "/" + form.loginPage;
                 String errorPage = form.errorPage.startsWith("/") ? form.errorPage : "/" + form.errorPage;
                 String landingPage = form.landingPage.startsWith("/") ? form.landingPage : "/" + form.landingPage;
+                String postLocation = form.postLocation.startsWith("/") ? form.postLocation : "/" + form.postLocation;
+                String usernameFieldName = form.usernameFieldName;
```

Comment:
@mahieddine Hi, it looks good, minor suggestion - the last 3 local vars are not conditionally initialized - they are just passed further to the mechanism - as such it would be simpler to use `form.usernameFieldName`, etc